hapi/build.gradle.kts

```diff
@@ -32,7 +32,7 @@ tasks.cloneHederaProtobufs {
     // uncomment below to use a specific tag
     //    tag = "v0.51.0"
     // uncomment below to use a specific branch
-    branch = "main"
+    branch = "remove-NodeGetInfo"
 }
```

Comment:
should this one be reverted back ?

Reply:
There is a four-step process required for protobuf updates currently to minimize the timeframe when develop might fail to compile.
1. Create branch in proto repo and update protobuf definitions there
1. Merge PR in services pointed to that branch with code changes
1. Merge PR in proto repo bringing the protobuf changes to main
1. Merge PR in services returning the develop branch to main.

We _could_ simplify the process if we pointed develop to a specific _commit_ on protobuf/main and never to the branch, but that isn't what is done currently.